modules/afmongodb/CMakeLists.txt

```diff
@@ -1,8 +1,12 @@
+if (ENABLE_MONGODB STREQUAL OFF)
```

Comment:
`ENABLE_MONGODB` should be evaluated as boolean. According to documentation `0, OFF, NO, FALSE, N, IGNORE, NOTFOUND, the empty string, or ends in the suffix -NOTFOUND` would equally evaluate to false.

You can just simply write
```
if (NOT ENABLE_MONGODB)
  return()
endif()
```

Reply:
This would be an incompatible change: currently when someone builds syslog-ng with cmake, it is not required to explicitly enable a module.

`(NOT ENABLE_MONGODB)` is `false` when you not define the `ENABLE_MONGODB` variable.

Reply:
Oh ok I see.